analysed-packages/nginx-http-auth-digest/version-1.0.0/nginx-http-auth-digest-1.0.0-SPDX2TV.spdx

```diff
@@ -0,0 +1,200 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_nginx-http-auth-digest_1.0.0.tar.gz_1725356707.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Mario Schuknecht
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of nginx-http-auth-digest 1.0.0.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-09-03T09:45:08Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: nginx-http-auth-digest_1.0.0.tar.gz
+PackageFileName: nginx-http-auth-digest_1.0.0.tar.gz
+SPDXID: SPDXRef-upload582
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 31f5aa9f1f2c3d8e043cd1d294571bef39162820
+PackageChecksum: SHA1: 43ca13f06cfa50bb69f0b7434ed0a91abb17320b
+PackageChecksum: SHA256: f09851e6309560a8ff3e901548405066c83f1f6ff88aa7171e0763bd9514762b
+PackageChecksum: MD5: 4ae8d157e623abf987234f819d0137a7
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-2-Clause
```

Comment:
Can you please correct the declared license to LicenseRef-BSD-2-Clause-c91da7fa6fc7de48c5e9b0395b97a6aa because this contains the correct text

Reply:
The package license declared was corrected.